code/modules/games/cards.dm

```diff
@@ -118,15 +130,17 @@
 
 	if(!H)
 		H = new(get_turf(src))
+		H.dir = NORTH
 		user.put_in_hands(H)
 
 	var/datum/playingcard/P = cards[1]
 	H.cards += P
 	cards -= P
 	H.parentdeck = src
+	H.update_values()
 	H.update_icon()
 	user.visible_message("<span class='notice'>[user] draws a card.</span>","<span class='notice'>You draws a card.</span>")
-	to_chat(user,"<span class='notice'>It's [P].</span>")
+	to_chat(user,"<span class='notice'>It's the [P].</span>")
```

Comment:
```suggestion
	to_chat(user,"<span class='notice'>It's \the [P].</span>")
```

Reply:
I'm not familiar with text macros, but since `P` is a datum, it just prints "It's Ace of Spaces." in chat.

Reply:
Ah, you might be able to use `\improper` on the name then I believe? I'm not too clear with text macros either.